### PR TITLE
Content for Shopistant 

### PR DESCRIPTION
**Pull request by ZarSaeed**

### CODE REVIEW

File: app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java [hunk 1]
```diff
@@ -48,76 +48,55 @@ public void initComponent(){
     public void onClick(View v){
         switch (v.getId()){
             case R.id.a_01:
```
Comment (Adeel91) [inline app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java]: It can be improved by using one case and then you can pass id e.g; **a_01, a_02** etc from another function like you did it in function **intent** to pass values but doesn't matter this is just complex to read and change in future, so for now we can just leave this issue and approve your pull request

**Review by Adeel91 — APPROVED** [review]
Some issues with coding standards but I guess this is your first time working with pull request etc so I'm going to approve it now.
By the way good job (thumbsup)

File: app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ */
+
+public class DetailsFragment extends Fragment {
+    private  int quantity=0;
+    private  String itemId;
+    private String aislePosition;
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view=inflater.inflate(R.layout.fragment_products,container,false);
```
Comment (Adeel91) [inline app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java:29]: this is not the best practice to supply everything in onCreateView because if we need to use the same below code somewhere then we cannot re-use it, so not a good approach

File: app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        aislePosition=getArguments().getString("AISLE");
+
+        final EditText edtName= (EditText) view.findViewById(R.id.fragment_product_name);
+        final EditText edtPrice= (EditText) view.findViewById(R.id.fragment_product_price);
+        final EditText edtQun= (EditText) view.findViewById(R.id.fragment_edt_product_qunatitiy);
+        Button btnInc= (Button) view.findViewById(R.id.fragment_btn_increase);
+        Button btnDec= (Button) view.findViewById(R.id.fragment_btn_decrease);
+        Button btnUpdate= (Button) view.findViewById(R.id.fragment_btn_update);
+
+        if (!name.equals("")&&!price.equals("")){
```
Comment (Adeel91) [inline app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java:43]: the if condition is not readable you should supply some spaces between two conditions

File: app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            edtName.setText(name);
+            edtPrice.setText(price);
+            edtQun.setText(""+quantity);
+        }
+
+        btnInc.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (quantity<=10){
```
Comment (Adeel91) [inline app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java:52]: if you read the PSR documentation then you will see that how and where you have to put spaces in condition.
well the condition should look like this ** if (quantity <= 10) { ** with spaces

File: app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+import android.widget.Button;
+
+public class MapActivity extends AppCompatActivity implements View.OnClickListener{
+    Button a01,a02,a03,a04,a05,a06,a07,a08,a09,a10,a11,a12,a13,a14;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_map);
+        initComponent();
+        a01.setOnClickListener(this);
```
Comment (Adeel91) [inline app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java:15]: this is not the way to setOnClickListener but it's alright for now

File: app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+        a07.setOnClickListener(this);
+        a08.setOnClickListener(this);
+        a09.setOnClickListener(this);
+        a10.setOnClickListener(this);
+        a11.setOnClickListener(this);
+        a12.setOnClickListener(this);
+        a13.setOnClickListener(this);
+        a14.setOnClickListener(this);
+    }
+    public void initComponent(){
```
Comment (Adeel91) [inline app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java:30]: there should be extra empty line in between the end of **onCreate** and **initComponent** functions

File: app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+                break;
+        }
+    }
+
+    public void intent(String data){
+        Intent intent=new Intent(MapActivity.this,ShelfsActivity.class);
+        intent.putExtra("AISLE",data);
+        startActivity(intent);
+    }
+
```
Comment (Adeel91) [inline app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java:100]: Extra blank lines should be removed according to PSR

File: app/src/main/java/com/example/zar/shopistantcontent/ShelfsActivity.java [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_shelfs);
+        Intent intent=getIntent();
+        aislePosition=intent.getStringExtra("AISLE");
+        initComponent();
+      /*  Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);*/
+
+        Log.e(TAG,aislePosition);
```
Comment (Adeel91) [inline app/src/main/java/com/example/zar/shopistantcontent/ShelfsActivity.java:44]: Do you really need this Log.e in the code???

File: app/src/main/res/layout/activity_map.xml [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+
+        android:layout_centerVertical="true"
+        android:layout_centerHorizontal="true">
+        <Button
+            android:id="@+id/a_01"
+            app:layout_gravity="left|top"
+            android:text="A01"
+            style="@style/Button"
+            android:textColor="@android:color/white"
+            android:layout_width="128.57dp"
```
Comment (Adeel91) [inline app/src/main/res/layout/activity_map.xml:26]: are you sure **128.57dp** is responsive for different screens??

File: app/src/main/res/layout/activity_shelfs.xml [hunk 10]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fitsSystemWindows="true"
+    tools:context="com.example.zar.shopistantcontent.ShelfsActivity">
+
+<!--    <android.support.design.widget.AppBarLayout
```
Comment (Adeel91) [inline app/src/main/res/layout/activity_shelfs.xml:10]: You must remove the commented lines from the code

File: app/src/main/res/values/arrays.xml [hunk 11]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```
Comment (Adeel91) [inline app/src/main/res/values/arrays.xml:2]: if there is nothing in this file then why did you add arrays.xml in your commit?

File: app/.gitignore [hunk 12]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (Adeel91) [inline app/.gitignore:1]: On the root folder of Shopistant, you already have .gitignore file then why did you add this again in app folder???

**Review by Adeel91 — COMMENTED** [review]
I've seen some major issues within your pull request, so please have a look!

**Comment by Adeel91** [comment]
One more important thing @ZarSaeed, what's the minimum SDK of Shopistant?

File: app/.gitignore (hunk 12 above)
Comment (ZarSaeed) [reply app/.gitignore:1]: it's in folder of ShopistantContent should i remove it ?

**Comment by ZarSaeed** [comment]
minimum SDK is 19 Should i decrease it?

File: app/src/main/res/values/arrays.xml (hunk 11 above)
Comment (ZarSaeed) [reply app/src/main/res/values/arrays.xml:2]: i removed this arrays.xml file.

File: app/src/main/java/com/example/zar/shopistantcontent/ShelfsActivity.java (hunk 8 above)
Comment (ZarSaeed) [reply app/src/main/java/com/example/zar/shopistantcontent/ShelfsActivity.java:44]: i just used it for check in purpose will remove it in next commit.

File: app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java (hunk 4 above)
Comment (ZarSaeed) [reply app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java:52]: i will do it with space in next commit Thanks for this.

File: app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java (hunk 2 above)
Comment (ZarSaeed) [reply app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java:29]: Will add some functions for it. Thanks for reviewing the code.

File: app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java (hunk 1 above)
Comment (ZarSaeed) [reply app/src/main/java/com/example/zar/shopistantcontent/MapActivity.java]: I should do an individual setOnClickListner for every view.

**Comment by ZarSaeed** [comment]
Thanks for reviewing the code.

File: app/src/main/res/values/arrays.xml (hunk 11 above)
Comment (AdeelSFarm) [inline app/src/main/res/values/arrays.xml]: If you're not using this file or retrieving anything from this file then it's useless and you can remove it and whenever you need it in future then just create it again

File: app/.gitignore (hunk 12 above)
Comment (AdeelSFarm) [inline app/.gitignore]: Yes remove it because we already have in root folder

File: app/src/main/res/values/arrays.xml (hunk 11 above)
Comment (Adeel91) [reply app/src/main/res/values/arrays.xml:2]: If you're not using this file or retrieving anything from this file then it's useless and you can remove it and whenever you need it in future then just create it again

File: app/.gitignore (hunk 12 above)
Comment (Adeel91) [reply app/.gitignore:1]: Yes remove it because we already have in root folder

**Comment by Adeel91** [comment]
You're welcome! I think if you will decrease the SDK then we can capture a big market of android.. minimum SDK 19 is too high I guess

**Comment by Adeel91** [comment]
and one last thing fix my comments in your next pull request (PR)
